spec/depctl_spec.rb

```diff
@@ -3,14 +3,18 @@
 depctl_file =
   File.expand_path(File.join(File.dirname(__FILE__), '../depctl/bin/depctl.rb'))
 load depctl_file
+require 'webmock/rspec'
+WebMock.allow_net_connect!
```

Comment:
Could we change the default to `disable_net_connect`?

Reply:
I had to allow net connect because of the docker registry gem. during load time it already makes a request :/